Python/Product/Cookiecutter/Commands/CreateFromCookiecutterCommand.cs

```diff
@@ -26,7 +26,7 @@ class CreateFromCookiecutterCommand : Command {
         }
 
         public override void DoCommand(object sender, EventArgs args) {
-            CookiecutterPackage.Instance.ShowWindowPane(typeof(CookiecutterToolWindow), true);
+            CookiecutterPackage.Instance.NewCookiecutterSession();
```

Comment:
Again, passing in an IServiceProvider or package instance is better than using the global.